packages/@react-spectrum/progress/docs/ProgressCircle.mdx

```diff
@@ -0,0 +1,103 @@
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/progress';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {ProgressCircle} from '@react-spectrum/progress';
+import {View} from '@react-spectrum/view';
+```
+
+# ProgressCircle
+
+<p>{docs.exports.ProgressCircle.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ProgressCircle']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/circle-loader/'}
+  ]} />
+
+## Example
+
+```tsx example
+<ProgressCircle aria-label="Loading…" value={50} />
+```
+
+## Value
+
+By default ProgressCircles are controlled with the `value` prop representing current percentage as an integer.
+In this case, the minimum and maximum values default to 0 and 100, respectively.
+
+```tsx example
+<ProgressCircle aria-label="Loading…" value={25} />
+```
+
+Alternatively, a different scale can be used by setting the `minValue` and `maxValue`.
+
+```tsx example
+<ProgressCircle aria-label="Loading…" minValue={50} maxValue={150} value={100} />
+```
+
+### Indeterminate
+[View guidelines](https://spectrum.adobe.com/page/circle-loader/#Indeterminate)
+
+By default, ProgressCircles are determinate. Use a determinate ProgressCircle when progress can be calculated against a specific goal.
+
+```tsx example
+<ProgressCircle aria-label="Loading…" value={50} />
+```
+
+Use an indeterminate ProgressCircle when progress is happening but the time or effort to completion can’t be determined.
+
+```tsx example
+<ProgressCircle aria-label="Loading…" isIndeterminate />
+```
+
+### Accessibility
+
+ProgressCircle requires an `aria-label` or `aria-labelledby` attribute.
+
+```tsx example
+<div>
+  <ProgressCircle aria-labelledby="pc-label" isIndeterminate />
+  <label id="pc-label" style={{display: 'block'}}>Loading…</label>
+</div>
+```
+
+If using the over background variant, ensure the background offers enough contrast for the ProgressCircle to be legible and meets color contrast guidelines.
```

Comment:
is there a difference in your reference to the over background variant here vs. when you referenced it on line 88 but put it in backticks that time?